src/Models/Assessment/Excel/CoreReport/Financial_Summary.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
P1: system.linq is not being used anywhere, it can be removed. You can identify the objects/classes that are not being used, as Visual Studio IDE makes them appear dull.